src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -682,10 +682,14 @@ private void handleLayers() {
             }
             handleRootCrsList(srs);
 
-            CloseableIterator<LayerInfo> layers;
-            layers = catalog.list(LayerInfo.class, filter);
+            // create layer groups filter
+            final Filter lgFilter = Predicates.acceptAll();
```

Comment:
A layer group can be part of a workspace, and a "namespace" filter actually filters on the workspace name, so, I'd factor in at least this bit.